tests/src/python/test_qgsvectorfilewriter.py

```diff
@@ -13,22 +13,24 @@
 __revision__ = '$Format:%H$'
 
 import qgis
+from qgis.core import (
+    QgsVectorLayer,
+    QgsFeature,
+    QgsGeometry,
+    QgsPoint
+)
 
-from qgis.core import (QgsVectorLayer,
-                       QgsFeature,
-                       QgsGeometry,
-                       QgsPoint
-                       )
+from qgis.testing import (
+    start_app,
+    unittest
+)
 
-from utilities import (getQgisTestApp,
-                       TestCase,
-                       unittest,
-                       writeShape
-                       )
-QGISAPP, CANVAS, IFACE, PARENT = getQgisTestApp()
+from utilities import writeShape
 
+start_app()
```

Comment:
Maybe this isn't relevant since you're just refactoring existing behavior, but shouldn't these be in an `if __name__ == "__main__"`